src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/ApprenticeController.cs

```diff
@@ -858,5 +862,19 @@ public async Task<IActionResult> DataLockRequestRestart(DataLockRequestRestartRe
 
             return View(viewModel);
         }
+
+        [Route("{apprenticeshipHashedId}/details/resend-email-invitation")]
+        [HttpGet]
+        public async Task<IActionResult> ResendEmailInvitation([FromServices] IAuthenticationService authenticationService, ResendEmailInvitationRequest request)
+        {
+            await _commitmentsApiClient.ResendApprenticeshipInvitation(request.ApprenticeshipId, new SaveDataRequest { UserInfo = authenticationService.UserInfo }, CancellationToken.None);
+
+            return RedirectToAction("ApprenticeshipDetails", new {
+                AccountHashedId = request.AccountHashedId,
+                ApprenticeshipHashedId = request.ApprenticeshipHashedId,
+                ShowNotification = true,
+                NotificationTitle = "The invitation email has been resent."

```

Comment:
hopefully these last 2 params will not be needed

Reply:
removed